force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -26,184 +26,164 @@
      * @param a The first number.
      * @param b The second number.
      * @return The absolute value of the difference
      */
     public static Double calculateDifferenceOfSquares(Double a, Double b) {
-        // Square the input numbers using the Math.pow() function
-
-        // Calculate the absolute difference using Math.abs() function
-
-        // Return the result
-        return null; // Replace null with the variable you used to store the result
+        Double doubleA = math.pow(a, 2);
+        Double doubleB = math.pow(b, 2);
+        Double result = math.abs(doubleA - doubleB);
+        return result;
     }
 
     /**
-     * Question 7
      * Generate a random number between 50 and 100
      * Example: generateRandomNumber() should return a number like 75.
      * @return A integer between 50 and 100.
      */
     public static Integer generateRandomNumber() {
-        // Use Math class to generate number between 50 and 100
-
-        // Return the random integer
-        return null; // Replace null with the variable you used to store the result
+        Integer randomNumber = (Integer)Math.random() + 51;
+        return randomNumber;
     }
 
     /**
-     * Question 8
      * Format the current date in the current user locale format.
      * Example: getCurrentDate() should return a string like "3/21/2001" for US locale users. Locale/String may differ depending on your location.
      * @return The current date in the current user locale format.
      */
     public static String getCurrentDate() {
-        // Get the todays date
-
-        // Format the current date
-
-        // Return the result
-        return null; // Replace null with the variable you used to store the result
+        Date myDate = Date.today();
+        String newDate = myDate.format();
+        return newDate;
     }
 
     /**
-     * Question 9
      * Given a date time return the day of the week in number format
      * @param dt the current date and time
      * @return The number of the day of the week for the given date.
      */
     public static String getDayOfWeek(DateTime dt) {
-        // Define a map of day of the week as a key and day number as the value
-        // Monday = 1, Tuesday = 2, etc.
-
-        // Get the day name using .format('EEEE');
-
-        // Get the day number from the map using the day name
-
-        // Return the result as a string
-        return null; // Replace null with the variable you used to store the result
+        Map<String, Integer> dayMap = new Map<String, Integer>();
+        dayMap.put('Monday', 1);
+        dayMap.put('Tuesday', 2);
+        dayMap.put('Wednesday', 3);
+        dayMap.put('Thursday', 4);
+        dayMap.put('Friday', 5);
+        dayMap.put('Saturday', 6);
+        dayMap.put('Sunday', 7);
+
+        String dayName = dt.format('EEEE');
+        Integer dayNumber = dayMap.get(dayName);
+        return String.valueOf(dayNumber); 
```

Comment:
How might you combine all three of these lines? Also, what could you do to your map to skip converting an Integer into a String?